py/desispec/redshiftqa/desi_per_fiber_qa_commands.sh

```diff
@@ -0,0 +1,14 @@
+# Compute the per-fiber redshift success rates and save them to disk
```

Comment:
bash scripts should go into bin/ or etc/, not under py/desispec/ .  bin/ is for scripts that are added to $PATH and could be run from the command line by any user from any directory; etc/ is for more specialized scripts or examples of what to run, but aren't included in what gets installed by default. I think this should go under etc/.

Reply:
All these scripts are moved to etc/.